app/src/shared/utils/storageNodes.js

```diff
@@ -0,0 +1,15 @@
+// Storage nodes are predefined and come from STORAGE_NODES env variable. It's a comma-separated
+// list of node names and addresses. Format: name0:address0,name1:address1.
+//
+// Spaces around names and addresses are stripped.
+
+const nodes = (process.env.STORAGE_NODES || '')
+    .split(/\s*,\s*/)
```

Comment:
Comma might not be the best separator in this case... I can image nodes being nodes named as eg. "Helsinki, Finland"

Reply:
You're goddamn right. I'll go with a `;`.